pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -142,6 +142,39 @@ def __call__(self, value, system):
             restriction_on_landownership['Reference'] = reference
             restriction_on_landownership['Article'] = article
 
+        # One restriction entry per theme
+        theme_restriction = {}
+        text_element = [
+            'Information', 'Lawstatus_Code', 'Lawstatus_Text', 'ResponsibleOffice_Name',
+            'ResponsibleOffice_OfficeAtWeb', 'SymbolRef', 'TypeCode'
+        ]
+        for restriction_on_landownership in extract_dict.get('RealEstate_RestrictionOnLandownership', []):
+            theme = restriction_on_landownership['Theme_Code']
+            if theme not in theme_restriction:
+                theme_restriction[theme] = restriction_on_landownership
+                for element in text_element:
+                    restriction_on_landownership[element] = set([restriction_on_landownership[element]])
```

Comment:
Almost there :) I just get a  KeyError: 'SymbolRef' on line 156 now I haven't had before

Reply:
I'll check on a new branch if that's related to our new data or also happening with the sample data

Reply:
Effectively, just fixed :-)